test/test_fsdp2/test_fsdp2_eager.py

```diff
@@ -76,16 +73,11 @@ def get_local_inp(self, dtype: torch.dtype = torch.float32):
         dist.broadcast(global_inp, src=0)
         return global_inp.view(self.world_size, -1)[self.rank].view(16, 16)
 
-    def swap_linear_with_dynamic(
-        self, module: nn.Module, use_float8_linear=False, **kwargs: Any
-    ) -> nn.Module:
-        if use_float8_linear:
-            kwargs["scaling_type_x"] = TensorScalingType.DYNAMIC
-            kwargs["scaling_type_w"] = TensorScalingType.DYNAMIC
-            kwargs["scaling_type_dL_dY"] = TensorScalingType.DYNAMIC
-            return swap_linear_with_float8_linear(module, Float8Linear, **kwargs)
-        else:
-            return swap_linear_with_float8_linear(module, Float8DynamicLinear, **kwargs)
+    def swap_linear_with_dynamic(self, module: nn.Module, **kwargs: Any) -> nn.Module:
```

Comment:
can we just remove this since this is the default?

Reply:
agreed in principle, but ideally that would be a separate PR since it's only tangentially related